src/obsidian/helpers/TldrawAppEmbedViewController.ts

```diff
@@ -9,7 +9,10 @@ export type OnChangeViewMode = (mode: ViewMode) => void;
  */
 export type ImageViewModeOptions = {
     bounds?: BoxLike;
-} & Omit<TldrawImageProps, 'bounds' | 'snapshot'>;
+} & Omit<TldrawImageProps, 'bounds' | 'snapshot'> & {
+    targetDocument?: Document;
+    assets?: any;
+};
```

Comment:
Revert, unless a good reason is needed for these fields to be included.

```suggestion
} & Omit<TldrawImageProps, 'bounds' | 'snapshot'>;
```